packages/Discretization/test/PointCloudProblemGenerator/create_ascii_data_from_exodus.py

```diff
@@ -0,0 +1,71 @@
+# This script reads and exodus file and dumps its node coordinates and
+# connectivities of all elements in all element blocks into 2 seperate files
+# to be used by driver test code
+
```

Comment:
Can you run pep8 for the python file

Reply:
what is pep8?

Reply:
In python you have an official style to follow so there are programs that check that your code follows the standard  https://pypi.python.org/pypi/pep8